packages/fxa-auth-server/test/local/routes/account.js

```diff
@@ -1327,7 +1327,7 @@ describe('/account/stub', () => {
     }
   });
 
-  it('rejects creating an account with an invalid email domain', async () => {
+  it('#integration - rejects creating an account with an invalid email domain', async () => {
```

Comment:
Why are these considered integration test?

Reply:
Let me double check on this. It either failed without infrastructure running (ie mysqldb) or it just took a really long time.

Reply:
Good catch. It didn't appear to need the integration tag.